Automattic/jetpack
# Gutenberg: Use Fullscreen mode in Calypsoify

#### Changes proposed in this Pull Request:

Calypsoify uses now the Gutenberg Fullscreen Mode instead of injecting the close button and including custom CSS.

We realized that the Gutenberg Fullscreen Mode is providing a UI very similar to Calypso, so by using it we can simplify the code and make it more maintainable. 

~This PR also includes a small fix for an 500 error raised when Jetpack is checking if Gutenberg is enabled.~ Fixed on #10389.

Fixes Automattic/wp-calypso#27937

#### Testing instructions:

* Edit a post in Gutenberg
* Append `&calypsoify=1` to the URL and reload the post editor.
* Make sure that the editor is displayed in Fullscreen Mode.
* Check that there are no options for disabling the Fullscreen Mode when clicking on the "More" button.
* Confirm that the back button uses now an icon and it redirects to the posts list in Calypso when clicking on it.

#### Screenshots:

Before
<img width="1275" alt="screen shot 2018-10-22 at 14 12 45" src="https://user-images.githubusercontent.com/1233880/47292141-68ba7c80-d606-11e8-9372-b9de36037b21.png">

After
<img width="1278" alt="screen shot 2018-10-22 at 15 17 53" src="https://user-images.githubusercontent.com/1233880/47294639-f9e12180-d60d-11e8-8031-b715aab319d2.png">

#### Proposed changelog entry for your changes:

I don't think this is needed given Calypsoify is mainly used internally. Anyway, if needed, this is my proposed changelog:

> Calypsoify uses now the Gutenberg Fullscreen Mode instead of injecting the close button and including custom CSS.
